### PR TITLE
Adding skill overlays and hotm stuff in pv + other minor stuff

### PR DESCRIPTION
**Pull request by nopothegamer**
https://github.com/NotEnoughUpdates/NotEnoughUpdates/blob/a62ebaaf51b8aee37631534009a7d2d944c8de6d/Update%20Notes/2.1.md

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -23,7 +23,7 @@ apply plugin: 'com.github.johnrengelman.shadow'
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
-version = "2.0-PRE"
```
Comment (IRONM00N) [inline build.gradle:26]: just change this to `2.1-REL`

File: src/main/java/io/github/moulberry/notenoughupdates/NEUEventListener.java [hunk 2]
```diff
@@ -240,7 +243,7 @@ public void onTick(TickEvent.ClientTickEvent event) {
             inventoryLoadedTicks = 3;
         }
 
-        if (Keyboard.isKeyDown(Keyboard.KEY_NUMPAD1) && Keyboard.isKeyDown(Keyboard.KEY_NUMPAD4) && Keyboard.isKeyDown(Keyboard.KEY_NUMPAD9)) {
+        if ((Keyboard.isKeyDown(Keyboard.KEY_NUMPAD1) && Keyboard.isKeyDown(Keyboard.KEY_NUMPAD4) && Keyboard.isKeyDown(Keyboard.KEY_NUMPAD9) || NotEnoughUpdates.INSTANCE.config.hidden.bHopBan)) {
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/NEUEventListener.java:246]: please don't add something called bhop, there will be some non on the forums complaining about it

File: src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java [hunk 3]
```diff
@@ -861,6 +862,14 @@ public void processCommand(ICommandSender sender, String[] args) {
         }
     });
 
+    SimpleCommand bHopCommand = new SimpleCommand("neubhop", new SimpleCommand.ProcessCommandRunnable() {
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/commands/Commands.java:865]: no

**Review by DeDiamondPro — CHANGES_REQUESTED** [review]
I'm in with iron that neubhop isn't a good idea because forum nerds

**Review by DeDiamondPro — APPROVED** [review]

**Review by IRONM00N — APPROVED** [review]

**Pull request merged**